constitution.tex

```diff
@@ -1,7 +1,15 @@
-\documentclass[10pt]{report}
+\documentclass[12pt]{report}
 \usepackage[utf8]{inputenc}
-
-\title{Constitution and Bylaws}
+\usepackage{titlepic}
+\usepackage{graphicx}
+\usepackage[pdftex,bookmarks, bookmarksnumbered=true, colorlinks=true,
+hypertexnames=false]{hyperref}
```

Comment:
Red is even worse, just get rid of `colorlinks=true`

Reply:
Oh i didn't realize it defaulted to red